library/oidc.c

```diff
@@ -502,7 +502,7 @@ static void ext_accept(uv_work_t *wr) {
         return;
     }
 
-    char buf[1024];
+    char buf[4096];
```

Comment:
some tokens are bigger than others... :)

Reply:
this does not get receive token